src/data-mgt/python/jobs/events.py

```diff
@@ -5,6 +5,7 @@
 
 DEVICE_REGISTRY_STAGING_URL = os.getenv("DEVICE_REGISTRY_STAGING_URL")
 DEVICE_REGISTRY_PRODUCTION_URL = os.getenv("DEVICE_REGISTRY_PRODUCTION_URL")
+ENVIRONMENT = os.getenv("ENVIRONMENT", "STAGING")
```

Comment:
Is this where we switch environments during tests?
Interested to learn about how the deployments for the different ENVs are handled.

Reply:
@Baalmart, there is no switching of environments, if the `ENVIRONMENT` variable is set to `PRODUCTION`, the measurements are sent to both staging and production else they are sent to staging.